DataFormats/PatCandidates/interface/Jet.h

```diff
@@ -177,6 +177,20 @@ namespace pat {
     void scaleEnergy(double fScale) override { scaleEnergy(fScale, "Unscaled"); }
     void scaleEnergy(double fScale, const std::string& level);
 
+    // Methods to work with a JER correction factor
+    //
+    // is the JER correction factor valid, i.e. has it been written using the saveJerFactor method
+    bool isJerFactorValid() const;
+
+    // load the currently saved JER correction factor and check whether the factor was propely set before loading
+    float loadJerFactor() const;
+
+    // save a JER correction factor
+    void saveJerFactor(float jerFactor_);
```

Comment:
```suggestion
    void setJerFactor(float jerFactor_);
```
as above for coding rules

Reply:
done